src/report/html.rs

```diff
@@ -0,0 +1,128 @@
+use assets;
+use errors::*;
+use ex::Experiment;
+use mime;
+use minifier;
+use report::{Comparison, CrateResult, ReportWriter, TestResults};
+use results::TestResult;
+use std::collections::HashMap;
+
+#[derive(Serialize)]
+enum Color {
+    Single(&'static str),
+    Striped(&'static str, &'static str),
+}
+
+trait ResultColor {
+    fn color(&self) -> Color;
+}
+
+impl ResultColor for Comparison {
+    fn color(&self) -> Color {
+        match self {
+            Comparison::Regressed => Color::Single("#db3026"),
+            Comparison::Fixed => Color::Single("#5630db"),
+            Comparison::Skipped => Color::Striped("#494b4a", "#555555"),
+            Comparison::Unknown => Color::Single("#494b4a"),
+            Comparison::SameBuildFail => Color::Single("#65461e"),
+            Comparison::SameTestFail => Color::Single("#788843"),
+            Comparison::SameTestSkipped => Color::Striped("#72a156", "#80b65f"),
+            Comparison::SameTestPass => Color::Single("#72a156"),
+            Comparison::Error => Color::Single("#d77026"),
+        }
+    }
+}
+
+impl ResultColor for TestResult {
+    fn color(&self) -> Color {
+        match self {
+            TestResult::BuildFail => Color::Single("#db3026"),
+            TestResult::TestFail => Color::Single("#65461e"),
+            TestResult::TestSkipped | TestResult::TestPass => Color::Single("#62a156"),
+            TestResult::Error => Color::Single("#d77026"),
+        }
+    }
+}
+
+#[derive(Serialize)]
+struct Context<'a> {
+    ex: &'a Experiment,
+    static_url: String,
```

Comment:
I'm fine with getting rid of this since it seems to be pointless. Up to you.

Reply:
Removed.